### ISSUE TITLE
'import type' -> swallows 'type' and breaks import

### ISSUE THREAD

**Issue by peritus** [Amwam/js-import-sort#21]
So, tried running `js-import-sort` against our large code base, and this happened:

``` diff
 -import type {
 -  NavigationSceneRendererProps
 -} from 'react-native/Libraries/NavigationExperimental/NavigationTypeDefinition';
 +import { NavigationSceneRendererProps } from 'react-native/Libraries/NavigationExperimental/NavigationTypeDefinition';
```

Notice that `import type` becomes `import`. Can you support `import type` as well ?

**Comment by Amwam** [comment]
Sorry this took a while to get too, I've released v1.1.0 which should allow for flow support now.

Thanks for reporting this!

**Comment by peritus** [comment]
@amwam thanks, works like a charm, will be using it!
